patients/repo.go

```diff
@@ -1091,16 +1109,8 @@ func (r *repository) TideReport(ctx context.Context, clinicId string, params Tid
 	}
 	tags := store.ObjectIDSFromStringArray(*params.Tags)
 
-	if params.CgmLastUploadDateFrom == nil || params.CgmLastUploadDateFrom.IsZero() {
-		return nil, errors.New("no lastUploadDateFrom provided")
-	}
-
-	if params.CgmLastUploadDateTo == nil || params.CgmLastUploadDateTo.IsZero() {
-		return nil, errors.New("no lastUploadDateTo provided")
-	}
-
-	if params.CgmLastUploadDateFrom.After(*params.CgmLastUploadDateTo) || params.CgmLastUploadDateFrom.Equal(*params.CgmLastUploadDateTo) {
-		return nil, errors.New("provided lastUploadDateFrom is after or equal to lastUploadDateTo")
+	if params.LastDataCutoff == nil || params.LastDataCutoff.IsZero() {
+		params.LastDataCutoff = pointer.FromAny(time.Now().UTC().AddDate(0, 0, -5))
```

Comment:
Why `-5`?

Reply:
Was not able to track the reason, so I asked product on intended behavior, and the goal is to make this field required with no default. It now returns an error when unset.